docs/src/main/asciidoc/key-values.adoc

```diff
@@ -0,0 +1,101 @@
+== Key Values
+
+Azure App Configuration Supports multiple types of key values, some of which have special features built into them.
+
+=== Placeholders
+
+The client library supports configurations with `${}`-style environment placeholders. If referencing a Azure App Configuration key with a placeholder any prefix needs to be removed in the reference. For example: `/application/config.message` is referenced as `${config.message}`.
```

Comment:
What are these configurations/environment variables used for? < answer this in a sentence after the first to make this clear. Ex: "These placeholders can be used to separate between environments such as prod, dev, and test, and let you easily switch between them"

Reply:
I think you misunderstand the use of the word environment here. It is environment as environment variable. The wording here is consistent with Springs own documentation on the subject. This isn't a feature we added, we just notifying customers it works exactly the same as in Spring Config and how to use it with our configurations.

Reply:
Could you add this information with maybe a link to where spring defines environmental variables?

Reply:
These are System Environment variables. Nothing to do with Spring besides that Spring uses them too.